lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
I think the only goal we want to achieve here is to avoid screen reader to to read this so it could be done by `aria-hidden="true"` but at same time we should make sure that in any case if the image is broken and alternative text should be presented to user so therefore `alt="XSeries Program Certificates"` is added.